spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -58,13 +66,49 @@
 				<artifactId>google-cloud-storage</artifactId>
 				<version>${google-cloud-storage.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>com.google.cloud</groupId>
+				<artifactId>google-cloud-trace</artifactId>
+				<version>${google-cloud-trace.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>com.google.cloud.trace.v1</groupId>
+				<artifactId>sink</artifactId>
+				<version>${cloud-trace-java.version}</version>
+				<exclusions>
+					<exclusion>
+						<groupId>io.grpc</groupId>
+						<artifactId>grpc-all</artifactId>
+					</exclusion>
+					<exclusion>
+						<groupId>com.google.protobuf</groupId>
+						<artifactId>protobuf-java</artifactId>
+					</exclusion>
+					<exclusion>
+						<groupId>com.google.guava</groupId>
+						<artifactId>guava</artifactId>
+					</exclusion>
+				</exclusions>
+			</dependency>
 
 			<dependency>
 				<groupId>commons-logging</groupId>
 				<artifactId>commons-logging</artifactId>
 				<version>1.2</version>
 			</dependency>
 
+			<dependency>
```

Comment:
Starter is enough - https://github.com/spring-cloud/spring-cloud-sleuth/blob/master/spring-cloud-starter-sleuth/pom.xml#L26-L29

Reply:
`spring-cloud-gcp-trace` depends only on the core and not the starter, our starter does depend on the spring-cloud-starter-sleuth though. Makes sense to keep both?

Reply:
IMO no. Either pick the starter or core. Cause the starter picks core transitively.

Reply:
Spoke w/ @marcingrzejszczak on hangout. This is the dependency management section so this was OK. `spring-cloud-gcp-trace` depends on core, and `spring-cloud-gcp-starter-trace` depends on starter